ipld/cid/src/lib.rs

```diff
@@ -105,121 +127,8 @@ impl<'de> de::Deserialize<'de> for Cid {
-    }
-
-    /// Create a new v0 CID.
-    pub fn new_v0(codec: Codec, hash: Multihash) -> Cid {
-        Cid::new(codec, Version::V0, hash)
-    }
-
-    /// Constructs a cid with bytes using default version and codec
-    pub fn new_from_cbor(bz: &[u8], code: Code) -> Self {
-        let hash = code.digest(bz);
-        Cid {
-            version: Version::V1,
-            codec: Codec::DagCBOR,
-            hash,
-        }
-    }
-
-    /// Create a new CID from raw data (binary or multibase encoded string)
-    pub fn from_raw_cid<T: TryInto<Cid>>(data: T) -> Result<Cid, T::Error> {
-        data.try_into()
-    }
-
-    /// Create a new CID from a prefix and some data.
-    pub fn new_from_prefix(prefix: &Prefix, data: &[u8]) -> Result<Cid, Error> {
-        let hash: Multihash = Code::try_from(prefix.mh_type)
-            .map_err(|e| Error::Other(e.to_string()))?
-            .digest(data);
-        Ok(Cid {
-            version: prefix.version,
-            codec: prefix.codec.to_owned(),
-            hash,
-        })
-    }
-
-    fn to_string_v0(&self) -> String {
-        use multibase::{encode, Base};
-
-        let mut string = encode(Base::Base58Btc, self.hash.to_bytes());
-
-        // Drop the first character as v0 does not know
-        // about multibase
-        string.remove(0);
-
-        string
-    }
-
-    fn to_string_v1(&self) -> String {
-        use multibase::{encode, Base};
-
-        encode(Base::Base32Lower, self.to_bytes().as_slice())
-    }
-
-    fn to_bytes_v0(&self) -> Vec<u8> {
-        self.hash.to_bytes()
-    }
-
-    fn to_bytes_v1(&self) -> Vec<u8> {
-        let mut res = Vec::with_capacity(32);
-        res.write_varint(u64::from(self.version)).unwrap();
-        res.write_varint(u64::from(self.codec)).unwrap();
-        // TODO avoid allocation here
-        res.append(&mut self.hash.to_bytes());
-
-        res
-    }
-
-    /// Returns encoded bytes of a cid
-    pub fn to_bytes(&self) -> Vec<u8> {
-        match self.version {
-            Version::V0 => self.to_bytes_v0(),
-            Version::V1 => self.to_bytes_v1(),
-        }
-    }
-
-    /// Returns prefix for Cid format
-    pub fn prefix(&self) -> Prefix {
-        Prefix {
-            version: self.version,
-            codec: self.codec.to_owned(),
-            mh_type: self.hash.code(),
-            mh_len: self.hash.size() as usize,
-        }
-    }
-
-    /// Returns cid in bytes to be stored in datastore
-    pub fn key(&self) -> Vec<u8> {
-        self.to_bytes()
-    }
-}
-
 impl fmt::Display for Cid {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let encoded = match self.version {
-            Version::V0 => self.to_string_v0(),
-            Version::V1 => self.to_string_v1(),
-        };
-        write!(f, "{}", encoded)
-    }
-}
-
-impl fmt::Debug for Cid {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "Cid(\"{}\")", self)
+        write!(f, "{}", self.0)
```

Comment:
This print is showing up as a debug print and it's really hard to read in certain logs, can you change this back to printing the actual `to_string`?